javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,19 @@
+const fullName = "Dennis Nguyen";
+let isCool = true;
+let totalPets = 4;
```

Comment:
These should be using `const` instead of `let`

Reply:
![Screenshot 2023-07-10 134129](https://github.com/dennis-pnguyen/c0723-code-solutions/assets/122412863/384f9cb1-ef89-45f8-bbc1-5f2ec15f84bb)